### PR TITLE
GITHUB-5455-red

### PR DESCRIPTION
**Pull request by a2xchip**
**Description (for Contributor and Core Developer)**

- Add validation error when no currencies and/or locales were set. 
- Add behat scenario
- Clean up behats

[See styleguide requirements](https://docs.akeneo.com/master/styleguide/index.php#Forms-AknFieldContainer)

image

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added Behats                      | OK
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: features/channel/edit_channel.feature [hunk 1]
```diff
@@ -65,11 +65,19 @@ Feature: Edit a channel
...
+    And I am on the "mobile" channel page
     When I change the "Locales" to "German (Germany)"
     And I press the "Save" button
     Then I should not see the text "There are unsaved changes."
     When I am on the locales page
     And I filter by "activated" with operator "equals" and value "yes"
     Then the grid should contain 1 elements
     And I should see locales "de_DE"
+
+  Scenario: Successfully display validation on NO_CONFLICT response
```
Comment (nidup) [inline features/channel/edit_channel.feature]: this scenario description is too technical imho, what are you testing from business point of view? you're testing "Successfully display validation message when the required currencies field is empty"

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/form/common/save.js [hunk 2]
```diff
@@ -99,6 +99,7 @@ define(
             fail: function (response) {
                 switch (response.status) {
                     case 400:
+                    case 409:
```
Comment (nidup) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/form/common/save.js]: why do we use 409 when editing a channel and 400 when editing the product? legacy stuff on product? (409 seems ok for me, not saying we need to change for channel or product but I try to understand why there is a difference)

File: features/acl/assert_acl.feature [hunk 3]
```diff
@@ -46,21 +46,14 @@ Feature: Define user rights
...
-    And I am on the "Administrator" role page
-    And I visit the "Permissions" tab
     When I revoke rights to resource <permission>
     And I save the role
     And I am on the <page> page
     Then I should not see the text "<button>"
 
     Examples:
       | permission                  | page                                         | button |
+      | Remove a channel            | "mobile" channel                             | Delete |
```
Comment (nidup) [inline features/acl/assert_acl.feature]: :+1: back to a more "standard" use

**Comment by nidup** [comment]
is it well working on required code when we create a new channel?

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/form/common/save.js (hunk 2 above)
Comment (a2xchip) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/form/common/save.js]: Found it :-) Changed

**Comment by nidup** [comment]
:tada:

**Comment by a2xchip** [comment]
:tada: :tada: :tada: :tada:

**Pull request merged**